jobrunner/config.py

```diff
@@ -110,9 +110,10 @@ class ConfigException(Exception):
 
 
 ACTIONS_GITHUB_ORG = "opensafely-actions"
+ACTIONS_GITHUB_ORG_URL = f"https://github.com/{ACTIONS_GITHUB_ORG}"
 ALLOWED_GITHUB_ORGS = (
     os.environ.get("ALLOWED_GITHUB_ORGS", "opensafely").strip().split(",")
-)
+) + [ACTIONS_GITHUB_ORG]
```

Comment:
I can see exactly why you'd want to do this, but unfortunately I don't think we can because we don't want to allow studies to be run from the `opensafely-actions` org. Action code has a slightly different risk profile from study code as it can only work on data that's already been extracted; it can't invoke cohortextractor to pull out more data. So we want to keep studies restricted to just the `opensafely` org and have a separate org for actions.

Reply:
I've squashed the commits on this branch, but I'm going to address this by not touching `ALLOWED_GITHUB_ORGS`.